Apps/WebClient/src/ClientApp/src/views/profile.vue

```diff
@@ -123,6 +123,19 @@ export default class ProfileView extends Vue {
         );
     }
 
+    private get LastLoginDateStrings(): string[] {
```

Comment:
We don't capitalize the first letter of getters in the front-end.

Nit: I'd suggest the name `formattedLoginDateTimes()` for the getter and `loginDateTimes` for the property.